addons/binding/org.openhab.binding.volvooncall/src/main/java/org/openhab/binding/volvooncall/internal/dto/Attributes.java

```diff
@@ -0,0 +1,82 @@
+/**
+ * Copyright (c) 2010-2019 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.volvooncall.internal.dto;
+
+import java.util.List;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+
+/**
+ * The {@link Attributes} is responsible for storing
+ * informations returned by vehicule attributes rest answer
+ *
+ * @author Gaël L'hopital - Initial contribution
+ */
+@NonNullByDefault
+public class Attributes {
+    public @NonNullByDefault({}) String vehicleType;
```

Comment:
I mean, a "dto" doesn't need to be "@NonNullByDefault", it should just help to prevent NPEs.
If you use "@NonNullByDefault({})" you are basically removing that annotation and disable the protection. Just remove it then.

Normally you would instead assign default values (to prevent NULL) or annotate with `@Nullable`